mobile/src/translations/en.json

```diff
@@ -32,7 +32,8 @@
       "accountDetails": {
         "noTransactions": {
           "title": "No transactions"
-        }
+        },
```

Comment:
just two languages?

Reply:
Yeah so far, but might try to import the languages from the other mobile app.